### DISCUSSION TITLE
3.0.2 version

### DISCUSSION THREAD

**Discussion by tonylu110** [tonylu110/uyou-todo-electron#43] [Announcements]
- Added Collapse text (note UI)
- Fixed use `My Today` only in normal mode

<hr /><em>This discussion was created from the release <a href='https://github.com/tonylu110/uyou-todo-electron/releases/tag/3.0.2'>3.0.2 version</a>.</em>

**Comment by I-icrous-I** [comment]
1.
![uyou_ToDo_bucIQQ3KI9](https://github.com/user-attachments/assets/ca13bc9c-85d8-4fee-8bfe-2e52e65ebfeb)
界面元素显示似乎出现了部分冲突。

2.
设置项中的“不显示关闭对话框”、“记住关闭窗口操作”和“最小化系统托盘”似乎无法正常工作。
无论是否启用这三项，点击关闭窗口，均为直接关闭。
关闭设置项“使用系统标题栏”并重启后，功能回复正常。

**Reply by tonylu110** [reply]
这些问题之前就纪录了，后续修复，这次主要是修复首次加载

**Reply by tonylu110** [reply]
[3.0.3 版本](https://github.com/tonylu110/uyou-todo-electron/releases/tag/3.0.3)，已经修复
